sdks/python/apache_beam/typehints/trivial_inference.py

```diff
@@ -376,6 +376,14 @@ def infer_return_type_func(f, input_types, debug=False, depth=0):
   inst_size = 2
   opt_arg_size = 0
 
+  # Python 3.10: bpo-27129 changes jump offsets to use instruction offsets,
+  # not byte offsets. The offsets were halved (16 bits fro instructions vs 8
+  # bits for bytes), so we have to double the value of arg.
+  if (sys.version_info.major, sys.version_info.minor) == (3, 10):
```

Comment:
should we use the 2 multiplier for all minor versions >= 10 ?

Reply:
Yes, I'll change it. When I wrote ` == (3, 10)`, I was thinking of `GEN_START` which was added in 3.10, but then removed in 3.11.

Reply:
Updated.